xbmc/pvr/PVRDatabase.cpp

```diff
@@ -977,34 +977,49 @@ bool CPVRDatabase::Persist(CPVRTimerInfoTag& timer)
 
   // insert a new entry if this is a new timer, or replace the existing one otherwise
   std::string strQuery;
+  const std::shared_ptr<CPVRTimerType> timerType = timer.GetTimerType();
+  unsigned int timerTypeId = timerType ? timerType->GetTypeId() : PVR_TIMER_TYPE_NONE;
```

Comment:
Only local timers (== reminders) should be persisted in the database. And the timer type of a reminder never must be nullptr. I fear we have a bug somewhere that leads to storing non-reminder timers in the db.